classes/plugins/ThemePlugin.php

```diff
@@ -16,6 +16,7 @@
 
 namespace PKP\plugins;
 
+use Illuminate\Support\Facades\Cache;
 use APP\core\Application;
```

Comment:
Not sure why this was needed?

Reply:
This [line](https://github.com/pkp/pkp-lib/blob/main/classes/plugins/ThemePlugin.php#L958) is failing when we don't import this. It's used for getting download statistics which is why I was able to replicate the issue, because I was testing the chart.js version upgrade on the download statistics graph.